source/features/quick-comment-edit.tsx

```diff
@@ -7,6 +7,33 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import isArchivedRepo from '../helpers/is-archived-repo';
 
+function addQuickEditButton(commentForm: Element): void {
+	commentForm.classList.add('rgh-edit-comment');
+
+	const comment = commentForm.closest('.js-comment')!;
+	if (select.exists('.rgh-quick-comment-edit-button', comment)) { // #5572
+		return;
+	}
+
+	const button = (
+		<button
+			type="button"
+			role="menuitem"
+			className={`timeline-comment-action btn-link js-comment-edit-button ${pageDetect.isDiscussion() ? 'js-discussions-comment-edit-button' : ''} rgh-quick-comment-edit-button`}
+			aria-label="Edit comment"
+		>
+			<PencilIcon/>
+		</button>
+	);
+
+	const dropdown = comment.querySelector('.timeline-comment-actions > details:last-child')!;
+	if (pageDetect.isIssue()) {
+		dropdown.previousSibling!.replaceWith(button); // Replace whitespace node in issue comments header
```

Comment:
![image](https://user-images.githubusercontent.com/46634000/162253072-d841d7b1-bc4d-4094-858d-652e2a2a7652.png)

This might be too brittle to be worth it. Alternatively we could remove the left margin on the edit button.

Reply:
Oh yeah they only need to remove one space and this line ends up removing the reaction picker.

> Alternatively we could remove the left margin on the edit button

This